docs/detections/add-exceptions.asciidoc

```diff
@@ -62,7 +62,7 @@ NOTE: When you create a new exception from an alert, exception conditions are au
 +  
 [NOTE] 
 =======
-Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields and might cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts,Troubleshooting type conflicts and unmapped fields>>.
+A warning displays for fields with conflicts. Using these fields and might cause unexpected exceptions behavior. Refer to <<rule-exceptions-field-conflicts,Troubleshooting type conflicts and unmapped fields>> for more information.
```

Comment:
The sentence `Using these fields and might cause unexpected exceptions behavior.` sounds like missing something after the `and` or maybe it is just should not be there? Something like `Using these fields might cause unexpected exceptions behavior.`?

Reply:
Yep, the extra "and" is a typo. Fixing it below - thanks!

```suggestion
A warning displays for fields with conflicts. Using these fields might cause unexpected exceptions behavior. Refer to <<rule-exceptions-field-conflicts,Troubleshooting type conflicts and unmapped fields>> for more information.
```